test/tests/unit/wire_protocol_test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const expect = require('chai').expect;
+const bson = require('bson');
+const wireProtocol2_6 = require('../../../lib/wireprotocol/2_6_support.js');
+const wireProtocol3_2 = require('../../../lib/wireprotocol/3_2_support.js');
+
+const MockPool = {};
```

Comment:
can we actually make this a `class MockPool { write() { ... } }` so we can potentially reuse it for other purposes in the future?